src/main/java/me/totalfreedom/totalfreedommod/command/Command_gcmd.java

```diff
@@ -11,46 +14,73 @@
 public class Command_gcmd extends FreedomCommand
 {
 
+    public static final List<String> BLOCKED_COMMANDS = Arrays.asList(
```

Comment:
This would really be better located in the config.yml as not all servers will likely want these restrictions, not to mention it would be helpful for Mark to be able to change these on the fly if needed.